src/components/MoviesList/MoviesList.js

```diff
@@ -1,6 +1,14 @@
 import React from 'react';
 import './MoviesList.scss';
+import PropTypes from 'prop-types';
+import { MovieCard } from '../MovieCard/MovieCard';
 
-export const MoviesList = () => (
-  <>Put the list here</>
+export const MoviesList = ({ movies }) => (
+  <div className="movies">
+    {movies.map(movie => <MovieCard {...movie} key={movie.imdbId} />)}
+  </div>
 );
+
+MoviesList.propTypes = {
+  movies: PropTypes.arrayOf(PropTypes.object).isRequired,
```

Comment:
you forgot to specify the prop type of imdbId (by the way, you can add prop types of all object properties in the PropTypes.shape inside of PropTypes.arrayOf, but it's not required)